sources/web/datalab/polymer/components/shared-styles/shared-styles.html

```diff
@@ -25,12 +25,6 @@
         --app-content-font-family: 'Roboto', 'Noto', sans-serif;
         --primary-text-color : var(--primary-fg-color);
       }
-      *, *::after, *::before {
```

Comment:
Yes. For some reason this interferes with the toast animation.
This wasn't very well thought out anyway, it's expensive and doesn't really add much other than theme switching, which isn't a frequent operation.